src/xdocs/config_misc.xml

```diff
@@ -1320,6 +1320,118 @@ void foo(String aFooString,
       </subsection>
     </section>
 
+    <section name="OrderedProperties">
+      <subsection name="Description" id="OrderedProperties_Description">
+        <p>Since Checkstyle 8.21</p>
+        <p>
+          Checks if property keys (left from equal sign) are sorted.
```

Comment:
please use the same summary as I proposed in for javadoc.

Reply:
I synced the docu.